### PR TITLE
#54  added include_package_data wich should allow to use requirements…

### LINKED ISSUE #54: Fix PyPi install
Getting error message when running : pip install BullETS

Error Message : 
Collecting BullETS
  Downloading https://files.pythonhosted.org/packages/f4/69/e2be5695bf338a1f60bba365f5c75267bfb32b3cbcab62955a82b3c983f2/BullETS-0.0.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\PHILIP~1\AppData\Local\Temp\pip-install-9qddr4_m\BullETS\setup.py", line 4, in <module>
        with open('requirements.txt') as f:
    FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in C:\Users\PHILIP~1\AppData\Local\Temp\pip-install-9qddr4_m\BullETS\

### PR DESCRIPTION
**Pull request by TristanFecteau**
….txt

Closes #54

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -20,5 +20,7 @@
     url='https://github.com/AlgoETS/BullETS',
     package_dir={'': 'src'},
     packages=find_packages(where='src'),
-    install_requires=requirements
+    install_requires=requirements,
+    package_data={'': ['requirements.txt']},
+    include_package_data=True
```
Comment (SamiMammouche) [inline setup.py]: We can remove this line now since we have `package_data`

Comment (TristanFecteau) [reply setup.py]: No, from what I've seen online we need both (https://github.com/aws-quickstart/taskcat/issues/143)

Comment (SamiMammouche) [reply setup.py]: It doesn't look necessary according to setuptools' documentation: 
https://setuptools.readthedocs.io/en/latest/userguide/datafiles.html

Comment (TristanFecteau) [reply setup.py]: fixed, you are right it does not seem necessary

**Review by SamiMammouche — APPROVED** [review]

**Pull request merged**